### ISSUE TITLE
Replace default `p` with something that doesn't conflict with `Kernel.p`.

### ISSUE THREAD

**Issue by andrewculver** [bullet-train-co/nice_partials#36]

**Comment by andrewculver** [comment]
I'm leaning toward `np`, despite the conflict with the original `np` usage. 😬

**Comment by andrewculver** [comment]
I think I could be convinced to just do `partial` and skip the brief options.

**Comment by andrewculver** [comment]
Here's a draft version of the README updated with this change, so we can see how it feels. I think it's better? https://github.com/bullet-train-co/nice_partials/blob/features/partial-instead-of-p/README.md

**Comment by kaspth** [comment]
Yeah, I'm quite partial (lol) to `partial`. I think it's better than `content` already, so let me do a PR before we swap other packages to e.g. `content` or what not.
